### PR TITLE
Added scalars

### PR DESCRIPTION
**Pull request by tomitrescak**
The proposal for the scalars documentation.

### CODE REVIEW

**Comment by helfer** [comment]
Thanks a lot for the PR! We use Javascript in all our examples, so it would be great if the Date example could use JavaScript as well, and not Typescript.

One other thing is that on line 78 there's a typo. It should say `Odd`, but it says `Date`.

File: source/apollo-server/scalars.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+```js
+scalar Date
+
+type MyType {
+   created: Date
+}
+```
+
+And here is the resolver (it is defined in Typescript, for javascript simply remove type definitions):
```
Comment (helfer) [inline source/apollo-server/scalars.md]: Please change it to JavaScript so all our documentation uses the same language.

File: source/apollo-server/scalars.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    oddValue: Odd
+}
+```
+
+And now the resolver:
+
+```js
+import { Kind } from 'graphql/type'
+
+Date: {
```
Comment (helfer) [inline source/apollo-server/scalars.md]: It says `Date`, but it should say `Odd`.

**Comment by helfer** [comment]
And could you add the JSON type as well? Here's the JavaScript for that
```
function parseJSONLiteral(ast) {
  switch (ast.kind) {
    case Kind.STRING:
    case Kind.BOOLEAN:
      return ast.value;
    case Kind.INT:
    case Kind.FLOAT:
      return parseFloat(ast.value);
    case Kind.OBJECT: {
      const value = Object.create(null);
      ast.fields.forEach(field => {
        value[field.name.value] = parseJSONLiteral(field.value);
      });

      return value;
    }
    case Kind.LIST:
      return ast.values.map(parseJSONLiteral);
    default:
      return null;
  }
}  

const resolvers = 
  JSON: {
    __parseLiteral: parseJSONLiteral,
    __serialize: value => value,
    __parseValue: value => value,
  },
};
```

**Comment by tomitrescak** [comment]
I have done the changes. I have also added some reference to external documents.

**Comment by helfer** [comment]
Cool, thanks a lot!

**Pull request merged**